packages/chocolate-doom/fix-tmpdir.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/chocolate-doom-chocolate-doom-3.0.1/src/m_misc.c b/chocolate-doom-chocolate-doom-3.0.1.mod/src/m_misc.c
+index 7e69da8..d18cc0b 100644
+--- a/chocolate-doom-chocolate-doom-3.0.1/src/m_misc.c
```

Comment:
Also.. you mostly need to chang line 3 and 4 to  
```
--- ./src/m_misc.c
```
And
```
+++ ./src/m_misc.c
```
.... That works for me usually after diff -uNr

Reply:
Thanks for the suggestion